src/userCLI/build.gradle

```diff
@@ -20,10 +20,9 @@ dependencies {
     implementation project(":app")
     implementation "com.offbytwo:docopt:0.6.0.20150202"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation "org.slf4j:slf4j-nop:1.7.26"
```

Comment:
Remove multiple binding as binding already exist in hint `build.gradle`.

Reply:
I have restore this library, it is needed for getting logs from console in `hint-deploy` thus ignoring multiple slf4j binding warning.